### ISSUE TITLE
Improve documentation on how to write a response body in Jetty 12

### ISSUE THREAD

**Issue by cowwoc** [eclipse/jetty.project#10293]
**Jetty version**
12.0.0

**Java version**
20

**Question**
What is the correct write to write a response body in Jetty 12 using the Response object? I saw `void write(boolean last, ByteBuffer byteBuffer, Callback callback)` but the Javadoc doesn't indicate what exactly it is writing (the body? the headers? something else?) and it's not clear how to best carry out this operation in a synchronous manner (for those of us who wants to stick with synchronous calls).

**Comment by cowwoc** [comment]
This is the best I could come up with but I'm hoping there is a cleaner solution:

```
CountDownLatch latch = new CountDownLatch(1);
response.write(true, StandardCharsets.UTF_8.encode(body), Callback.from(latch::countDown,
throwable ->
{
	throw RuntimeException(throwable);
}));
latch.await();
```

**Comment by joakime** [comment]
There are a few ways to write response body.
The `Response.write()` methods will write the body (and commit the response, by writing headers if not written yet)

Also of note, the `Response` is a `Content.Sink`, so check out the static methods there.

See some example code snippets at 

https://github.com/eclipse/jetty.project/blob/jetty-12.0.x/documentation/jetty-documentation/src/main/java/org/eclipse/jetty/docs/programming/server/HandlerDocs.java

Eg: you an use an OutputStream if you wanted, or a JDK Flow API, or a bunch of other options.

**Comment by cowwoc** [comment]
@joakime Thank you. By the way, I think you have a bug at https://eclipse.dev/jetty/javadoc/jetty-12/org/eclipse/jetty/server/Handler.html

The Javadoc contains this code snippet:

```
response.write(true, callback, "hello");
```

As far as I can tell, there is no such method.

Also, https://eclipse.dev/jetty/javadoc/jetty-12/org/eclipse/jetty/server/Request.Handler.html#handle(org.eclipse.jetty.server.Request,org.eclipse.jetty.server.Response,org.eclipse.jetty.util.Callback) contains `ByteBuffer.wrap("Hello World\n".getBytes(StandardCharsets.UTF_8))`. I suggest replacing this with `StandardCharsets.UTF_8.encode("Hello World\n")` which is shorter and easier to read.

**Comment by sbordet** [comment]
Fixed by #10434.
